scripts/spcgeonode/django/initialize.py

```diff
@@ -87,10 +87,19 @@
     authorization_grant_type='authorization-code',
     skip_authorization=True
 )
-redirect_uris = [
-    'http://{}/geoserver'.format(os.getenv('HTTPS_HOST',"") if os.getenv('HTTPS_HOST',"") != "" else os.getenv('HTTP_HOST')),
-    'http://{}/geoserver/index.html'.format(os.getenv('HTTPS_HOST',"") if os.getenv('HTTPS_HOST',"") != "" else os.getenv('HTTP_HOST')),
-]
+_host = os.getenv('HTTPS_HOST',"") if os.getenv('HTTPS_HOST',"") != "" else os.getenv('HTTP_HOST')

```

Comment:
Https host does use the full https://url iirc. Therefore the chance is you get a duplicate for the protocol

Reply:
Checked it. It is not set with protocol. Therefore this is fine